### PR TITLE
Add visual demo to Accounts section

### PR DESCRIPTION
**Pull request by 0xGabi**

### CODE REVIEW

**Review by jley81 — APPROVED** [review]

**Review by wackerow — CHANGES_REQUESTED** [review]
@0xGabi Thanks for posting these to extend the use of Austin's video explanations =) If you wouldn't mind just fixing up the links so they can be embedded we'll try to get this up soon. Thanks!

File: src/content/developers/docs/accounts/index.md [hunk 1]
```diff
@@ -128,6 +128,14 @@ For dapp development, you'll want access to dummy accounts with test ETH so you
 
 An account is not a wallet. A wallet is the keypair associated with a user-owned account, which allow a user to make transactions from or manage the account.
 
+## A visual demo {#a-visual-demo} 
+
+Watch Austin walk you through hash functions, and key pairs. 
+
+<iframe width="100%" height="315" src="https://www.youtube.com/watch?v=QJ010l-pBpE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<iframe width="100%" height="315" src="https://www.youtube.com/watch?v=9LtBDy67Tho" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
[the remark this answers is not in the document]
Comment (0xGabi) [reply src/content/developers/docs/accounts/index.md]: Ops, good catch, thanks!

**Review by wackerow — APPROVED** [review]
@0xGabi looks good 👍 thanks again!

**Comment by wackerow** [comment]
@all-contributors please add @0xGabi for content

**Pull request merged**